sentry_sdk/integrations/strawberry.py

```diff
@@ -1,6 +1,15 @@
 import hashlib
-from functools import cached_property
 from inspect import isawaitable
+
+try:
+    from functools import cached_property
+except ImportError:
+    # The strawberry integration requires Python 3.8+. functools.cached_property
+    # was added in 3.8, so this check is technically not needed, but since this
+    # is an auto-enabling integration, we might get to executing this import in
+    # lower Python versions, so we need to deal with it.
+    cached_property = None  # type: ignore
```

Comment:
Why not raise `DidNotEnable` exception here, with a warning that the Python version is too low?

Reply:
That's a good idea. Had to move the `functools` import until after the `sentry_sdk` import that imports `DidNotEnable`, but it's still a nicer solution.